src/test/java/bio/terra/common/stairway/MonitoringHookTest.java

```diff
@@ -20,31 +20,62 @@
 import bio.terra.stairway.Step;
 import bio.terra.stairway.StepResult;
 import bio.terra.stairway.StepStatus;
-import io.opencensus.tags.TagValue;
-import io.opencensus.trace.SpanContext;
-import io.opencensus.trace.Tracing;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 @Tag("unit")
 public class MonitoringHookTest {
+  private static final Duration METRICS_COLLECTION_INTERVAL = Duration.ofMillis(10);
+  private OpenTelemetry openTelemetry;
+  private TestMetricExporter testMetricExporter;
+
+  @BeforeEach
+  void setup() {
+    testMetricExporter = new TestMetricExporter();
+    openTelemetry = openTelemetry(testMetricExporter);
+  }
 
-  private static final List<TagValue> ERROR_COUNT_TAGS =
-      List.of(
-          TagValue.create(FlightStatus.ERROR.name()),
-          TagValue.create(ErrorSpanRecordingFlight.class.getName()));
+  public OpenTelemetry openTelemetry(TestMetricExporter testMetricExporter) {
+    var sdkMeterProviderBuilder =
+        SdkMeterProvider.builder()
+            .registerMetricReader(
+                PeriodicMetricReader.builder(testMetricExporter)
+                    .setInterval(METRICS_COLLECTION_INTERVAL)
+                    .build());
+
+    var propagators = ContextPropagators.create(W3CTraceContextPropagator.getInstance());
+    return OpenTelemetrySdk.builder()
+        .setPropagators(propagators)
+        .setMeterProvider(sdkMeterProviderBuilder.build())
+        .build();
+  }
 
   @Test
   public void spansConnected() throws Exception {
     List<SpanContext> contextRecord = new ArrayList<>();
     RecordContextStep.setRecord(contextRecord);
```

Comment:
I know this is old code, but a setter here is unnecessary as an inner class's `private` fields aren't private with respect to its containing class. I would remove the setter and assign the field directly.
```suggestion
    RecordContextStep.contextRecord = contextRecord; 
```

It might be clearer to move `RecordContextStep.contextRecord` so it's a field on `MonitoringHookTest`. ~Once that's done it no longer needs to be `static` anymore either.~ Actually it has to be static somewhere because steps and flights must be static classes.